website/content/docs/upgrading/upgrade-to-1.10.x.mdx

```diff
@@ -31,3 +31,19 @@ Vault storage to an Etcd v3 cluster prior to upgrading to Vault 1.10.
 All storage migrations should have
 [backups](/docs/concepts/storage#backing-up-vault-s-persisted-data)
 taken prior to migration.
+
+ ### OTP Generation Process
+
+ Customers passing in OTPs during the root token generation process must modify
+ the OTP generation to be 2 characters longer before upgrading so the OTP can be 
+ xor'ed with the encoded root token. This change is due to service tokens in 1.10
+ being prefixed with "hvs." as opposed to "s.". 
+
+ ## Token Format Change
+
+ Token prefixes have been updated to be more easily identifiable. Service tokens that
```

Comment:
Maybe a bulleted list would be more suitable here? Will live that up to you:

Token prefixes were updated to be more easily identifiable. 

- Service tokens previously started with s. now start with hvs.
- Batch tokens that previously started with b. now start with hvb.
- Recovery tokens previously started with r. now start with hvr.

Additionally, non-root service tokens are now longer than before. Previously, service tokens were 26 characters; they now have a minimum of 95 characters. However, existing tokens will still work.